src/orchard/directory.clj

```diff
@@ -0,0 +1,67 @@
+(ns orchard.directory
+  "A port of BaseDirectories.java in soc/directories-jvm.
+  https://github.com/soc/directories-jvm"
+  {:author "Masashi Iizuka"
+   :added "0.5.0"}
+  (:require [clojure.java.io :as io]
+            [clojure.string :as str]
+            [orchard.misc :as u])
```

Comment:
I think `misc` is a better alias than `u`. :-) The weird aliases in the codebase today are due to the fact that this namespace was named `util` in cider-nrepl (if I recall correctly that is).